resources/seqr_metadata_templates/README.md

```diff
@@ -0,0 +1,275 @@
+# Organising rare disease data and metadata for seqr <!-- omit from toc -->
+
+![CPG Logo](../images/cpg_logo_1280x329.png)
+
+## Table of Contents <!-- omit from toc -->
+
+---
+
+- [1. Genomic Data](#1-genomic-data)
+- [2. Pedigree\_template](#2-pedigree_template)
+- [3. Families\_metadata\_template (Optional)](#3-families_metadata_template-optional)
+- [4. Individuals\_metadata\_template](#4-individuals_metadata_template)
+- [5. Sample\_mapping\_template](#5-sample_mapping_template)
+
+---
+
+### i. Purpose
+
+The purpose of this document is to provide instructions on how to prepare project specific data and metadata for use in seqr<br /> related to an individual's family history (pedigree).
+
+<br />
+
+### ii. Background
+
+The CPG utilises four distinct metadata files to provide information about samples to the variant curation team as they <br /> perform variant analysis in seqr.
+
+The four files are described in Table 1.
+
+A template for each of these files is provided in section iii. Quick Links, and instructions for filling out each template<br /> are included in this document.
+<br />
+<br />
+
+#### **Table 1:** Definitions of the metadata files used in rare disease projects <!-- omit from toc -->
+
+|     **Metadata template**    |     **Required**    |     **Description**    |
+|:---:|:---:|:---|
+|     *Pedigree_template*    |     **Yes**    |     Template file used to describe the individuals in each dataset <br />and how they relate to other individuals in <br />the same dataset, mainly their parents.<br /><br />The information in this file is used to generate<br /> the participant pedigrees*.    |
+|     *Families_metadata_template*    |     No    |     Template file used to describe the families in each dataset. |
+|     *Individuals_metadata_template*    |     **Yes**    |     Template file used to describe the clinical information <br />related to individuals in each dataset. |
+|     *Sample_mapping_template*    |     **Yes**    |     Template file used to map individual IDs AND sample IDs<br /> back to the files that have been transferred. |
+
+*A pedigree is a structured description of the phenotypical and familial relationships between samples.
+
+The CPG uses the tool ‘GATK HaplotypeCaller’, which can incorporate pedigree information in the genomic analysis of samples.
+
+<br />
+
+### iii. Quick Links
+
+1. [Excel templates](excel_templates/)<br />
+
+2. [csv templates](csv_templates/)<br />
+
+3. [tsv templates](tsv_templates/)<br />
```

Comment:
Do we need tsv and csv or should we just pick one (presumably csv for multi-line escaping in notes fields)?

Reply:
I agree it's probably not necessary to have both. I'm happy to just reduce it to csv